cmd/hydration-controller/main.go

```diff
@@ -82,6 +82,13 @@ func main() {
 		klog.Fatalf("Failed to register OpenCensus views: %v", err)
 	}
 
+	// Set up env var k8s.container.name for OC agent
```

Comment:
Add a comment about why we're doing this, like "Add the k8s.container.name resource label so that the google cloud monitoring and monarch metrics exporters will use the k8s_container resource type."

Maybe also copy that comment to the other places that k8s.container.name is being set.

Reply:
Done